start.sh

```diff
@@ -16,6 +15,14 @@ MEMCACHED="${MEMCACHED:-$( echo "${MEMCACHED_LINKED_NOTCP:-127.0.0.1}" )}"
 
 DOMAIN="${DOMAIN:localhost}"
 
+# Wait for postgresql rediness
+until psql -h "${PGHOST}" -U "postgres" -c '\l'; do
```

Comment:
This won't work in all cases, because you are hard-coding the user, and access to super-user postgres might not be available.

It might make sense to export PGUSER, PGHOST, PGPORT, PGPASSWORD above, and check
```psql "${PGDB}"``` instead, so that they will be used by the postgres client by default.

Also, just in case, for backwards-compatibility, how about we make this check non-default, move it into an environment variable and set it only in docker-compose file?

E.g. smth like

```bash
if [[ "${WAIT_FOR_POSTGRES}" = "1" ]]; then
    # the code to wait for postgres
fi
```

and then set WAIT_FOR_POSTGRES=1 in the docker-compose file?